docs/CS2113T-F09-3-Eggventory-UG.md

```diff
@@ -443,3 +495,8 @@ Matric No. | Matriculation number of student to be added
 Stock | A physical asset to be tracked by Eggventory  
 Stock Code | A unique string of characters to identify a stock  
 Stock Type | A category of stock
+
+
+[3.1-viewing-help-help]: #3.1-viewing-help:-help
+
+[3. Features and command usage  ]: #3.-features-and-command-usage
```

Comment:
Is this an error? i have removed this in #382

Reply:
yeah sry